Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Download ubuntu image with python and pip
+FROM nitincypher/docker-ubuntu-python-pip
+
+RUN apt-get update && \
+    apt-get install -y sudo && \
```

Comment:
It prevents running subsequent commands if a dependency fails to install -- basically it helps to fail fast if anything goes wrong. I can remove the && chain if you don't think it's necessary.

Reply:
Are you sure Docker doesn't already break when a run command fails?  My impression from some quick googling is that it does, but I didn't test it.

I think it would be cleaner to split the commands, however, the `apt-get update` and `apt-get install` should still be combined as per https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

Reply:
Removed the && chain except for `apt-get update` combined with `apt-get install`